projects/opentrain/gtfs_pipeline/heatmap/HeatMap.java

```diff
@@ -0,0 +1,325 @@
+package projects.opentrain.gtfs_pipeline.heatmap;
+
+import com.projects.opentrain.gtfs_pipeline.Protos.Stop;
+import com.projects.opentrain.gtfs_pipeline.Protos.StopTime;
+import com.projects.opentrain.gtfs_pipeline.Protos.Trip;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HeatMap {
+
+
+    public List<Stops_around> stopsAround = new ArrayList<>();

```

Comment:
Please use 2-space indentation.
You can use this to format your code:
`./fix-formatting.sh`